### PR TITLE
Add additional info to the emitter in order to expose it to the api. …

### PR DESCRIPTION
**Pull request by kgriesh**
…Adding baseUrl and route info  allows for collecting data on urls that contain path variables. I accept and agree to be bound by the terms of the IBM Contributor License Agreement.

### CODE REVIEW

**Comment by sjanuary** [comment]
The `bundleDependencies` section in package.json is needed by a downstream dependency

File: package.json [hunk 1]
```diff
@@ -12,9 +12,6 @@
     "semver": "^5.3.0",
     "jszip": "2.5.x"
   },
-  "bundleDependencies": [
-    "tar"
```
Comment (sam-github) [inline package.json]: tar is needed to run the install script. npm doesn't guarantee that a packages deps are installed before its install script is run, so the only safe way for an install script to use any deps is to have them be bundled. Whether any particular install sees this or not depends on race conditions, so while not bundling the install script's deps may work most of the time, for most people, it fails for us in CI, where we do lots of runs, across lots of systems.

**Comment by sjanuary** [comment]
Please remove the change to package.json and add an update to README.md to describe the extra properties on an http event.  A test for the new properties in https://github.com/RuntimeTools/appmetrics/blob/master/tests/probes/http-probe-test.js would also be nice to have.

**Comment by tobespc** [comment]
@kgriesh Hi, please can you update your pull request with the comments made by @sjanuary above.  We can then accept this request.  thanks

**Comment by koushikr** [comment]
@kgriesh Can you follow and resubmit a merge please?

Waiting for one.

**Comment by stalleyj** [comment]
@kgriesh This extra data is express specific, please can you explain the requirement to have these individually rather than just the url which is already part of the http event ?

**Comment by stalleyj** [comment]
@kgriesh Closing this PR as we haven't heard from you, please re-open if this is something you are still interested in.

**Pull request closed without merging**